source/talk/TemplateSlides.tex

```diff
@@ -0,0 +1,28 @@
+\documentclass{beamer}
+
+\setbeameroption{hide notes}
+
+\title{Presentation title}
+\author{Author}
+\institute{Affiliation}
+\date{\today}
+
+\begin{document}
+
+\frame{\titlepage}
+
+\begin{frame}
+\frametitle{Research question}
+\begin{itemize}
+	\item Point One 
+	\note{Notes on Main point}
+	\begin{itemize}
+		\item Sub Point
+		\item Sub Point  
+		\note{Notes on subpoint}
+	\end{itemize}
+ 	\item Point Two
+\end{itemize}
+\end{frame}
```

Comment:
@ew487 can we add an example transition slide between talk sections roughly along the lines of the below?

(Even better, but not essential, if these can be linked to the use of a `\section{}` designation.}

![image](https://github.com/JMSLab/Template/assets/7840353/5abb5157-beb0-45e0-a258-83ba91308bb4)

Reply:
Added [here](https://github.com/JMSLab/Template/pull/89/commits/57caa68034b01a423174e09db52d50ccc68c7d8e)!